.readthedocs.yml

```diff
@@ -1,7 +1,11 @@
-type: sphinx
-configuration: doc/sphinx-apidoc/conf.py
-python:
-    version: 3
-    requirements_file: requirements.txt
-formats:
-    - none
+version: 2
+
+sphinx:
+  builder: html
+  configuration: doc/sphinx-apidoc/conf.py
+
+conda:
+  environment: doc/environment.yml
```

Comment:
Shouldn't this be renamed to conda_environment.yml?

Reply:
I think it shouldn't. Using the default name (which to the best of my knowledge is just `environment.yml`) also makes sure that you don't have to specify anything special when creating the environment:
```bash
cd doc
conda env create
```